crates/red_knot_workspace/src/site_packages.rs

```diff
@@ -158,7 +135,7 @@ mod tests {
     // This test is skipped on Windows until we commit a Windows venv.
     #[cfg(not(target_os = "windows"))]
```

Comment:
```suggestion
    #[cfg_attr(ignore="Windows has a different venv layout", not(target_os = "windows"))]
```

I think it should then always compile the code (which is nice when doing refactoring) and even explicitly emit a message why the test is ignored on windows.

Reply:
Ah, clever!